mofa/simulation/lammps.py

```diff
@@ -8,9 +14,103 @@ class LAMMPSRunner:
     Args:
         lammps_command: Command used to launch LAMMPS
     """
+    lammps_command = "npt_tri"
+    lmp_sims_root_path = "lmp_sims"
+    cif_files_root_path = "cif_files"
+    cif_files_paths = []
 
-    def __init__(self, lammps_command: str):
-        self.lammps_command: str = lammps_command
+    def __init__(self, lammps_command: str = "npt_tri", lmp_sims_root_path: str = "lmp_sims", cif_files_root_path: str = "cif_files"):
+        """Read cif files from input directory, make directory for lammps simulation input files
+
+        Args:
+            lammps_command: lammps simulation type, default: "npt_tri"
+            lmp_sims_root_path: output directory, default: "lmp_sims"
+            cif_files_root_path: input directory to look for cif files: "cif_files"
+        Returns:
+            None
+        """
+        self.lammps_command = lammps_command
+        self.lmp_sims_root_path = lmp_sims_root_path
+        print("Making LAMMPS simulation root path at: " + os.path.join(os.getcwd(), self.lmp_sims_root_path))
```

Comment:
Use logging instead of print statements so it's easier to capture when it was called and suppress if unneeded.